generateBins.py

```diff
@@ -81,6 +131,11 @@ def process(configuration, gisPath = ""):
 
     return [ pfprRanges, zoneTreatments ]
 
+def classify(value, breaks):
```

Comment:
As a matter of personal style I tend to place helper functions like this below the function that they are used in.